src/backend/prisma/migrations/20230702055640_init/migration.sql

```diff
@@ -0,0 +1,30 @@
+-- CreateTable
+CREATE TABLE "project" (
+    "id" TEXT NOT NULL,
+    "name" VARCHAR(30) NOT NULL,
+    "description" VARCHAR(100) NOT NULL,
+    "url" VARCHAR(50) NOT NULL,
+    "lastMaintenance" TIMESTAMP(3),
```

Comment:
Do you need timestamp(3)? main difference from what I can tell is milliseconds, are they required?

Reply:
to tell you the truth you are commenting on a migration that was auto-generated by Prisma. 

this is the commit I sent you, you can see there how I configured my schema (that later on generated this migration):
https://github.com/Sasporta/projects-manager/pull/1/commits/0665d890ca5509ebe9151194247b40ca600ee442